package.json

```diff
@@ -25,20 +25,26 @@
   },
   "homepage": "https://github.com/Lemoncode/simplechart#readme",
   "dependencies": {
+    "@zeit/next-css": "^0.2.0",
     "@material-ui/core": "^3.0.1",
     "@zeit/next-sass": "^0.2.0",
     "@zeit/next-typescript": "^1.1.0",
     "babel-polyfill": "^6.26.0",
+    "css-loader": "^1.0.0",
     "d3": "^5.6.0",
     "d3-composite-projections": "^1.2.0",
     "express": "^4.16.3",
     "isomorphic-unfetch": "^2.1.1",
     "next": "^6.1.1",
+    "postcss-loader": "^3.0.0",
     "qs": "^6.5.2",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
+    "react-tag-input": "^5.1.2",
+    "style-loader": "^0.23.0",
     "topojson-client": "^3.0.0",
     "topojson-simplify": "^3.0.2",
+    "webpack": "^3.0.0",
```

Comment:
We don't need to install webpack. `next` has webpack under the hood

Reply:
I installed webpack, because when I tried to run the project It throws me this error;

```
module.js:549
    throw err;
    ^

Error: Cannot find module 'webpack/lib/Chunk'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\daily\Desktop\TFM\simplechart\node_modules\extract-text-webpack-plugin\dist\index.js:17:14)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```

Reply:
@Nasdan What can I do about it?